testsuite/pytests/test_aeif_cond_alpha_multisynapse.py

```diff
@@ -147,12 +149,13 @@ def test_single_multi_synapse_equivalence(self, have_plotting):
                 _ax.legend()
 
             ax[-1].semilogy(multisynapse_neuron_vm.get("events")["times"], error, label="errror")
-            fig.savefig("test_aeif_cond_alpha_multisynapse.png")
+            fig.savefig(os.path.join(os.environ.get("REPORTDIR", ""), "test_aeif_cond_alpha_multisynapse.png"))
```

Comment:
In previous discussions, I think `pathlib` has come up as preferred for path manipulation on strings.

Reply:
Manipulation via os.path would still work, but I agree, that unifying on `pathlib` seems cleaner to me. In many cases it allows more precise type annotations, for example (`: Path` instead of `: str`).
```suggestion
            fig.savefig(Path(os.environ.get("REPORTDIR", ".")) / "test_aeif_cond_alpha_multisynapse.png")
```